pathwar/pathwar
# fix: Add a fallback client route for not found/created routes (e.g.: missions), adjust menu active style

## What does this PR change?
It creates a default route in client routing to be returned when routes not created/non-existent yet are accessed.
It applies an active style to menu items when the user navigates.
Fix leaking background outside client app.
Minor refactoring: Turn wrapper into a functional component using hooks.

## How to test?
Open the app and try to access a route like `Missions`, you should see a message informing you the page is not available yet.

![image](https://user-images.githubusercontent.com/6627610/156052679-503019b5-6bd2-44e3-8169-999837105335.png)
